docs/proposals/algorithms/single-task-learning/fpn.md

```diff
@@ -0,0 +1,177 @@
+# Single task learning: FPN
+
+It is a traditional learning pooling all data together to train a single model. It typically includes a specialist model laser-focused on a single task and requires large amounts of task-specific labeled data, which is not always available on early stage of a distributed synergy AI project. 
```

Comment:
``It`` here may not be clear to readers. Maybe it would be better to directly refer ``single task learning`` here.

Reply:
Polished. See commit: https://github.com/kubeedge-sedna/ianvs/pull/4/commits/0bf938cf1f25680b6caa457ba2fbf6df3b7b1ec6